locust/.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__/
```

Comment:
Ah, one more thing, `venv` and `*.pyc` should be ignores here too like the `flask` directory.